src/components/mgt-people/mgt-people.ts

```diff
@@ -90,6 +90,28 @@ export class MgtPeople extends MgtTemplatedComponent {
   })
   public people: IDynamicPerson[];
 
+  /**
+   * allows developer to define queries of people for component
+   * @type {string[]}
+   */
+
+  @property({
+    attribute: 'people-query',
```

Comment:
How do you feel about calling this `people-queries` / `peopleQueries` instead since it's an array of queries?